### DISCUSSION TITLE
beginner vscode extension devloper

### DISCUSSION THREAD

**Discussion by atocodes** [microsoft/vscode-discussions#108] [Extension Development QnA] [answered]
Hi my name is BEAMLAK TADESSE and recently I've made an algorithm that converts the Arabian number to an Ethiopian (geez) number and I really want to make it a VsCode extension and but even tho I can code js but I am new to extension development for VsCode and I wonder if someone helps me to figure out how can I target the line numbers in the editor tab?

**Comment by gjsjohnmurray** [comment, marked as answer]
I don't think there's any [API](https://code.visualstudio.com/api/references/vscode-api) with which you can replace these line numbers or override their appearance. But you may be able to use the editor decorator API to prefix each line with a before-decorator showing its geez line number. See https://github.com/microsoft/vscode-extension-samples/tree/main/decorator-sample for an example of how to use this API.
